sphinx/writers/latex.py

```diff
@@ -565,7 +566,8 @@ def visit_document(self, node: Element) -> None:
             self.first_document = 0
         elif self.first_document == 0:
             # ... and all others are the appendices
-            self.body.append('\n\\appendix\n')
+            self.body.append(CR)
+            self.body.append('\\appendix' + CR)
```

Comment:
Side note: I thought adding `println()` (print line) helper as a alias of `self.body.append(s + CR)`. But I feel it's too much. So I withdrew the idea and use simple concatenation.